.changeset/lucky-hornets-invite.md

```diff
@@ -0,0 +1,64 @@
+---
+"@aws-amplify/ui-react-storage": minor
+"@aws-amplify/ui-react": minor
```

Comment:
Not sure if we need to bump ui-react?

Reply:
+1

Reply:
Good point. IM JUST WORRIED ABOUT NOT PUBLISHING PACKAGES